src/components/InRequest/InRequestEditPanel.tsx

```diff
@@ -93,17 +93,19 @@ export const InRequestEditPanel: FunctionComponent<IInRequestEditPanel> = (
     watch,
     reset,
     setValue,
+    getValues,
     register,
   } = useForm<any>();
   const updateRequest = useUpdateRequest(props.data.Id);
 
   // Setup watches
-  const empType = watch("empType");
-  const isNewCivMil = watch("isNewCivMil");
   const hasExistingCAC = watch("hasExistingCAC");
   const eta = watch("eta");
   const employee = watch("employee");
 
+  // Get values of fields that can't be changed so that UI can render based on them
+  const [empType, isNewCivMil] = getValues(["empType", "isNewCivMil"]);
+
```

Comment:
They sure are...  I initially second guessed myself on this, but looking again, it will render correctly when the data becomes available from React Query.  Fixed to use props directly.